components/form/form.js

```diff
@@ -77,6 +77,26 @@ class Form extends FormMixin(LitElement) {
 		this._submitData(submitter);
 	}
 
+	resetValidation() {
```

Comment:
Up for debate: should forms listen for the `d2l-dialog-close` event and automatically reset validation? That would solve the consumer case that came up (and presumably future similar cases), but obviously there's a risk that there are forms inside dialogs that wouldn't want this. We could flag that portion of the change to help mitigate that. Just trying to avoid everyone needing to call this when dialog's close if we could handle it for them.

Reply:
I think I could get onboard with that. That's probably the behaviour I would want, and if the edge-case arose, we could provide a way to opt out of that.

Reply:
I like that idea. Will update